flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/rel/LogicalWindowAggregate.scala

```diff
@@ -18,8 +18,6 @@
 
 package org.apache.flink.table.plan.logical.rel
 
-import java.util
```

Comment:
In principle I agree, but I it caused some building problem. Is there a practical reason for not importing the List class directly rather than creating "half references"?